dmvutil.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python3
```

Comment:
It would be nice if this dmv library code could be shared between the installer and other uses (e.g. driver tool and xsconsole). There isn't really anything installer-specific about most of it.